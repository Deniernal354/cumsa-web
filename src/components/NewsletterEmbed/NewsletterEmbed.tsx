export default function NewsletterEmbed() {
  return (
    <div className="rounded-xl overflow-hidden shadow ring-1 ring-black/10 bg-white p-4">
      <iframe
        src="https://embeds.beehiiv.com/1b1e2e2b-2e2e-4e2e-8e2e-2e2e2e2e2e2e?slim=true"
        style={{ width: "100%", minHeight: 320, border: "none", background: "transparent" }}
        title="Newsletter Signup"
        loading="lazy"
        allowFullScreen
      />
    </div>
  );
}
