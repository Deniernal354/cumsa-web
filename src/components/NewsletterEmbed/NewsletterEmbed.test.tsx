import { render } from "@testing-library/react";
import NewsletterEmbed from "./NewsletterEmbed";

test("NewsletterEmbed snapshot", () => {
  const { asFragment } = render(<NewsletterEmbed />);
  expect(asFragment()).toMatchSnapshot();
});
