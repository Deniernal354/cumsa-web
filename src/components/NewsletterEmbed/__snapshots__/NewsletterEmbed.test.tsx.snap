// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsletterEmbed snapshot 1`] = `
<DocumentFragment>
  <div
    class="rounded-xl overflow-hidden shadow ring-1 ring-black/10 bg-white p-4"
  >
    <iframe
      allowfullscreen=""
      loading="lazy"
      src="https://embeds.beehiiv.com/1b1e2e2b-2e2e-4e2e-8e2e-2e2e2e2e2e2e?slim=true"
      style="width: 100%; min-height: 320px; background: transparent;"
      title="Newsletter Signup"
    />
  </div>
</DocumentFragment>
`;
